corehq/apps/locations/util.py

```diff
@@ -117,26 +118,38 @@ def get_location_data_model(domain):
 
 class LocationExporter(object):
 
-    def __init__(self, domain, include_consumption=False, headers_only=False, async_task=None):
+    def __init__(self, domain, include_consumption=False, root_location_id=None,
+                 headers_only=False, async_task=None):
         self.domain = domain
         self.domain_obj = Domain.get_by_name(domain)
         self.include_consumption_flag = include_consumption
-        self.headers_only = headers_only
         self.data_model = get_location_data_model(domain)
         self.administrative_types = {}
         self.location_types = LocationType.objects.by_domain(domain)
         self.async_task = async_task
         self._location_count = None
         self._locations_exported = 0
 
+        if headers_only:
+            self.base_query = SQLLocation.objects
```

Comment:
```suggestion
            self.base_query = SQLLocation.objects.none()
```
(this was `.none()` before the move)